EGI-CLI-Swift.md

```diff
@@ -0,0 +1,61 @@
+# Using fedcloud and openstack slients
+
+## First install required Python packages
+
+```
+conda create -n egi python
+pip install openstackclient
+pip install fedcloudclient
+```
+
+You should be able to issue:
+```
+fedcloud token check
+```
+
+But you don't have a token yet.
+
+## Connect to EGI
+
+Go to https://aai.egi.eu/token/ to optain your client token. Copy your access token, and then set it:
+
+```
+export OIDC_ACCESS_TOKEN=<your_token>
+fedcloud token check
+```
+
+Last command should return a valid token.
+
+## Use Openstack
+
+You need to set other env variables to simplify things:
+
+```
+export EGI_SITE=CESNET-MCC
+export EGI_VO=vo.pangeo.eu
+```
```

Comment:
This is about to change in an upcoming release of `fedcloudclient` so I would rather leave the CLI options below.
```suggestion
```